src/main/java/com/clubobsidian/dynamicgui/DynamicGui.java

```diff
@@ -275,7 +277,7 @@ else if(this.proxy == Proxy.REDIS_BUNGEE)
 	private void registerListeners() 
 	{
 		this.eventManager.registerEvents(new com.clubobsidian.dynamicgui.listener.EntityClickListener());
-		this.eventManager.registerEvents(new com.clubobsidian.dynamicgui.listener.InventoryClickListener());
+		this.eventManager.registerEvents(new com.clubobsidian.dynamicgui.listener.InventoryInteractListener());
```

Comment:
Codacy found an issue: [Unnecessary use of fully qualified name 'com.clubobsidian.dynamicgui.listener.InventoryInteractListener' due to existing same package import 'com.clubobsidian.dynamicgui.*'](https://app.codacy.com/manual/virustotalop/DynamicGui/pullRequest?prid=5724759)